src/frontend/src/modals/IOModal/components/chatView/newChatView.tsx

```diff
@@ -89,7 +89,7 @@ export default function ChatView({
       return new Date(a.timestamp).getTime() - new Date(b.timestamp).getTime();
     });
 
-    if (messages.length === 0 && !lockChat && chatInputNode) {
+    if (messages.length === 0 && !lockChat && chatInputNode && isTabHidden) {
       setChatValue(chatInputNode.data.node.template["input_value"].value ?? "");
     } else if (isTabHidden) {
       setChatValue("");
```

Comment:
What about this `else if`?

It seems if the tab is hidden it sets the value to an empty string.

Reply:
@ogabrielluiz this is to set the startup value of empty string on chat, but only if user did not changed the tab on browser.

Reply:
before that everytime the user changes the tab from browser, the chat input loses Its value

Reply:
Yes, but I found a new scenario where the bug occurs again. On the first message of the chat, when there is a value in the template, if you type something and then switch tabs, the template value always reappears.

To prevent this behavior, we should only use the template value if the tab hasn't been changed. Since the default initial value of isTaChange is False, this is why we need this validation.